app/assets/scripts/components/traces/index.js

```diff
@@ -398,7 +398,7 @@ class Traces extends React.Component {
           </td>
           <td>{getUTCDate(trace.recordedAt)}</td>
           <td>{trace.ownerDisplayName}</td>
-          <td>{trace.length} m</td>
+          <td>{convertMeter2Kilometer(trace.length).length} {convertMeter2Kilometer(trace.length).unit}</td>
```

Comment:
We should maintain consistency in the usage of this function. In here it is used in the view, but for single traces it is applied in the reducer. I would implement both in the views, instead of in the reducers.